bin/deepforge-user

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+const program = require('../node_modules/webgme-engine/src/bin/usermanager').main(process.argv);
```

Comment:
`require('webgme-engine/src/bin/usermanager')` would be better than relative import.

Reply:
Is it ok if we use it directly, or should we mention `webgme-engine` as a dependency in `package.json` as well? Its not that it will matter much because `webgme-engine` is going to be in `node_modules` regardless.